packages/api/lib/testUtils.js

```diff
@@ -483,9 +483,9 @@ const createProviderTestRecords = async (context, providerParams) => {
 
   const insertPgRecord = await translateApiProviderToPostgresProvider(originalProvider);
   await providerModel.create(originalProvider);
-  const [providerCumulusId] = await providerPgModel.create(testKnex, insertPgRecord);
+  const [pgProvider] = await providerPgModel.create(testKnex, insertPgRecord);
```

Comment:
Same comment as previous. These changes are totally correct and pretty surprising. It seems like this would cause our providers not to be indexed in ES properly?